<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* 文本框和按钮 */
    background-color: rgb(173, 216, 230);
    QLineEdit, QPushButton {
        border: 2px solid #00008B;  /* 深蓝色边框 */
        background-color: #00008B;  /* 深蓝色背景 */
        color: #FFFFFF;             
        border-radius: 20px;  /* 更明显的圆角 */
        padding: 5px;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>9</y>
      <width>653</width>
      <height>498</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
        QTabWidget::pane {
            border-top: -1px;
            border-left: 1px solid lightgray;
            border-right: 1px solid lightgray;
            border-bottom: 1px solid lightgray;
        }
        QTabBar::tab {
            color: rgb(255, 255, 255);
            border: 0.5px solid lightgray;
            padding: 5px;
        }
        QTabBar::tab:selected {
            color: rgb(255, 255, 255);
            margin-bottom: -1px;
        }
       </string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">background-color: transparent; color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>Login</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacerLeft">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QListView" name="imageView">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>225</width>
            <height>173</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
              border-image: url(:/picture/logo.png);
              background-size: contain; /* 使图像适应控件大小 */
              background-repeat: no-repeat;
              background-position: center;
             </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerRight">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Telecharge DataBase&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>628</width>
          <height>34</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black; text-align: center;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="placeholderText">
         <string>Tele-Number</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>628</width>
          <height>34</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black; text-align: center;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Secret</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>628</width>
          <height>33</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>628</width>
          <height>34</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>register</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">background-color: transparent; color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Welcome to use Our Telecharge System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>119</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="styleSheet">
      <string notr="true">background-color: transparent; color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>Register</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="idCardEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black; text-align: center;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="placeholderText">
         <string>ID Card</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black; text-align: center;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="teleNumberEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black; text-align: center;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="placeholderText">
         <string>Tele-Number</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="secretEdit">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black; text-align: center;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Secret</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="registerButton">
        <property name="styleSheet">
         <string notr="true">border: 1px solid blue; color: black;
font: 75 9pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>User</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Server</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Admin</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QMenu {
      background-color: rgb(26, 26, 46);
      gridline-color: rgb(15, 52, 96);
      selection-background-color: rgb(15, 52, 96);
      color: rgb(255, 255, 255);
      font: 500 11pt &quot;Poppins&quot;;
     }
     QMenuBar {
      background-color: rgb(26, 26, 46);
      selection-background-color: rgb(15, 52, 96);
      color: rgb(255, 255, 255);
      spacing: 3px;
     }
     QMenuBar::item:selected {
      background: rgb(15, 52, 96);
      color: rgb(255, 255, 255);
     }
    </string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qrc/background.qrc"/>
 </resources>
 <connections/>
</ui>
